module/settings.js

```diff
@@ -1,4 +1,153 @@
+import DG from "./config.js";
+
+const { ApplicationV2, HandlebarsApplicationMixin } = foundry.applications.api;
+
+const SettingForm = class extends HandlebarsApplicationMixin(ApplicationV2) {
+  constructor() {
+    super();
+  }
+
+  static _namespace;
+
+  static get namespace() {
+    return this.constructor._namespace
+  };
+
+  static get settings() {
+    return {}
+  }
+
+  static init() {
+    const settings = this.settings;
+    for (const setting of Object.keys(settings)) {
+      game.settings.register(DG.ID, setting, {
+        scope: "world",
+        config: false,
+        name: game.i18n.localize(`DG.Settings.${setting}.name`),
+        hint: game.i18n.localize(`DG.Settings.${setting}.hint`),
+        ...settings[setting],
+      });
+    }
+  }
+
+  static DEFAULT_OPTIONS = {
+    tag: "form",
+    id: `${DG.ID}-setting-form`,
+    classes: [DG.ID, "settings-menu"],
+    window: {title:  ``, resizable: true},
```

Comment:
```suggestion
    window: {resizable: true},
```
Set title in each subclass (see below)